CHANGELOG.md

```diff
@@ -1,19 +1,30 @@
 # Changelog
 
+## 0.50.0
+
+### Changes
```

Comment:
I usually list anything that wasn't a bugfix that most people won't notice, so we can also include https://github.com/microsoft/vscode-pull-request-github/issues/1193 along with a screen shot.